src/network_manager.rs

```diff
@@ -72,6 +72,9 @@ impl PeerNetManager {
             InternalTransportType::from_transport_type(transport_type, self.peer_db.clone())
         });
         transport.stop_listener(addr)?;
+        let mut peer_db = self.peer_db.write();
```

Comment:
Listeners are thread that only accept connections. Stoping one of them just allow to stop a specific port that listen to new connection but doesn't close all connections with all the other peers

Reply:
You can check https://github.com/massalabs/massa/discussions/3149 to be more familair with architecture